docs/core/release_notes.md

```diff
@@ -0,0 +1,55 @@
+# 🚨 Breaking Changes for UiPath Python SDK (v2.2.0+)
+
+**Release Date:** November 30, 2025
+
+Version 2.2.0 of the **UiPath Python SDK** introduces the following breaking changes
+
+## Breaking Changes
+
+### 1. Minimum Python Version: 3.11+ Required
+
+**What's changing:** Python 3.10 is no longer supported for `uipath-python` and `uipath-langchain-python`
+
+**Action required:** Upgrade to Python 3.11 or higher.
+
+### 2. Import Path Change
+
+**What's changing:** The `UiPath` class has moved from `uipath` to `uipath.platform`.
+
+**Action required:** Update your imports:
+
+```python
+# Before
+from uipath import UiPath
+from uipath.models import Job, Asset, Queue
+from uipath.models import Entity 
+
+# After
+from uipath.platform import UiPath
+from uipath.platform.orchestrator import Job, Asset, Queue
+from uipath.platform.dataservice import Entity
+client = UiPath(...)
+```
+
+### 3. Transition to LangChain v1
+
+
+Review and update your code according to the [LangChain v1 Migration Guide](https://docs.langchain.com/oss/python/migrate/langchain-v1).
+
+## Migration Guide
+
+To avoid these breaking changes and keep your current setup, pin your dependency in `pyproject.toml`:
+
+```toml
+"uipath>=2.1.x,<2.2.0"
+```
+
+To migrate to v2.2.0+, please consider to:
+
+1. Upgrade your Python version to 3.11+ in `pyproject.toml`.
```

Comment:
We should mention the `pyproject.toml` needs to have `requires-python = ">=3.11"` (the version we have installed on our serverless image, otherwise `uv` will download the latest version)